game-core/src/main/java/games/strategy/engine/GameEngineVersion.java

```diff
@@ -34,7 +34,7 @@ public static GameEngineVersion of(final Version version) {
   public boolean isCompatibleWithEngineVersion(final Version engineVersion) {
     checkNotNull(engineVersion);
 
-    return version.withMicro(0).equals(engineVersion.withMicro(0));
+    return version.withPoint(0).equals(engineVersion.withPoint(0));
```

Comment:
With 3 numbers, and build number as the 3rd, it will be the 'point' version. I think this should then be 'withMinor', and you can remove the 'point' variable from Version.java.

build number was added in a way that the code was totally unaware of, it's happy splitting on periods and ignoring extra numbers. I don't think we want engine compat to depend on any change, 'minor' is actually pretty in-line with how we've been using it.

Reply:
Regarding your comment:
> This is actually going to cause problems as the 'point' number becomes the build number

I don't think it will cause any problems as I have coded it. I'm pretty sure I've tested all reasonable scenarios, although it was over a month now.